#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_fix-apiversion.dpatch by Thomas Weber <thomas.weber.mail@gmail.com>
##
## DP: Switching from 2.9.4 to 2.9.5, upstream changed 'aptversion' to
## DP: 'api_version' everywhere except in configure.in. This results in
## DP: non-apiversioned m- and oct-directories.

@DPATCH@
diff -urNad octave2.9-2.9.5~/configure.in octave2.9-2.9.5/configure.in
--- octave2.9-2.9.5~/configure.in	2006-03-22 23:25:58.000000000 +0100
+++ octave2.9-2.9.5/configure.in	2006-04-19 13:01:06.603336800 +0200
@@ -77,7 +77,7 @@
 OCTAVE_SET_DEFAULT(fcnfiledir, '$(datadir)/octave/$(version)/m')
 OCTAVE_SET_DEFAULT(localfcnfiledir, '$(datadir)/octave/site/m')
 OCTAVE_SET_DEFAULT(localapifcnfiledir,
-  '$(datadir)/octave/site/$(apiversion)/m')
+  '$(datadir)/octave/site/$(api_version)/m')
 OCTAVE_SET_DEFAULT(localverfcnfiledir, '$(datadir)/octave/$(version)/site/m')
 OCTAVE_SET_DEFAULT(localfcnfilepath,
   '$(localverfcnfiledir)//$(sepchar)$(localapifcnfiledir)//$(sepchar)$(localfcnfiledir)//')
@@ -93,7 +93,7 @@
 OCTAVE_SET_DEFAULT(localoctfiledir,
   '$(libexecdir)/octave/site/oct/$(canonical_host_type)')
 OCTAVE_SET_DEFAULT(localapioctfiledir,
-  '$(libexecdir)/octave/site/oct/$(apiversion)/$(canonical_host_type)')
+  '$(libexecdir)/octave/site/oct/$(api_version)/$(canonical_host_type)')
 OCTAVE_SET_DEFAULT(localveroctfiledir,
   '$(libexecdir)/octave/$(version)/site/oct/$(canonical_host_type)')
 OCTAVE_SET_DEFAULT(localoctfilepath,
